routers/repo/release.go

```diff
@@ -247,6 +247,7 @@ func EditRelease(ctx *context.Context) {
 	ctx.Data["title"] = rel.Title
 	ctx.Data["content"] = rel.Note
 	ctx.Data["prerelease"] = rel.IsPrerelease
+	ctx.Data["draft"] = rel.IsDraft
```

Comment:
Should be `ctx.Data["IsDraft"]`

If you feel like it, you might as well fix the above ones as well. Though not related to this issue :slightly_smiling_face: